Source/Core/VideoBackends/Vulkan/SwapChain.cpp

```diff
@@ -39,7 +41,20 @@ SwapChain::~SwapChain()
 
 VkSurfaceKHR SwapChain::CreateVulkanSurface(VkInstance instance, void* hwnd)
 {
-#if defined(VK_USE_PLATFORM_WIN32_KHR)
+#if defined(HAVE_GLFW)
+  GLFWwindow* window = static_cast<GLFWwindow*>(hwnd);
+  VkSurfaceKHR surface;
+
+  VkResult res = glfwCreateWindowSurface(instance, window, nullptr, &surface);
+  if (res != VK_SUCCESS)
+  {
+    LOG_VULKAN_ERROR(res, "glfwCreateWindowSurface failed: ");
```

Comment:
No, the `LOG_VULKAN_ERROR()` macro (actually `LogVulkanResult()`) converts res into a string and displays that.